sources/test/user/data.test.js

```diff
@@ -0,0 +1,195 @@
+const {beforeTests,afterTests, apolloServer, PERMISSION_DENIED_TEST} = require("../testHelper");
+
+// Pre and Post Test Scripts
+beforeAll(beforeTests);
+afterAll(afterTests);
+
+describe('Results: Users Queries and Mutations', () => {  
+
+  const { query, mutate } = apolloServer("a8mjiKYtt0PefnS524",["users.all","users.Auth","users.byName","users.byId","users.Update","users.Delete"]);
+  let testUser={
+    name:"Harish",
+    username:"HarishTeens"  
+  };
+  it('Get all Users', async () => {    
+      const GET_USERS = `
+        {
+          users{
+            ... on User{
+              name
+            }
+          }
+    
+        }
+      `;
+
+      const response = await query({ query: GET_USERS });
+      expect(response.data.users).toEqual([]);
+    });
```

Comment:
This essentially can be two tests
1. Input data to db and see if the response is what you expect
2. Check if the array has size 0

Together, it will check if, data is getting returned and it is an array of expected size

Reply:
I just removed this test as there is no use for this one. 
the first point you mentioned is done in the Auth user test